how-to/integrate-with-sso/integrate-with-okta/package.json

```diff
@@ -0,0 +1,42 @@
+{
+	"name": "openfin-workspace--integrate-with-okta",
+	"version": "10.3.0",
+	"description": "OpenFin Workspace -- Integrate with SSO - Okta",
+	"main": "index.js",
+	"scripts": {
+		"dos": "npm --prefix ../../common/ run dos",
+		"build": "npm run build-server & npm run build-client",
+		"build-server": "tsc --project ./server",
+		"build-client": "webpack build --config ./client/webpack.config.js --mode=development",
+		"build-client-prod": "webpack build --config ./client/webpack.config.js --mode=production",
+		"start": "npm run server",
+		"client": "npm --prefix ../../common/ run client",
+		"server": "node ./server/build/index.js",
+		"kill": "npm --prefix ../../common/ run kill",
+		"setup": "cd ../../../ && npm install && cd how-to/integrate-with-sso/integrate-with-okta && npm run build && npm --prefix ../../common/ run build",
+		"validate": "npm --prefix ../../../ run validate"
+	},
+	"author": "ameet.jayawant@openfin.co",
+	"license": "SEE LICENSE IN LICENSE.MD",
+	"dependencies": {
+		"@openfin/core": "^28.72.17",
+		"@openfin/workspace": "10.3.10",
+		"@openfin/workspace-platform": "10.3.10"
+	},
+	"devDependencies": {
+		"@okta/okta-auth-js": "^7.2.0",
```

Comment:
We aim to strip out all packages that are not used. I think this may be dependencies from another package.json as there doesn't seem to be references to these in the server or client side project. Can we strip it down to only the needed packages please (keeps our package lock smaller and reduces vulnerability by only bringing in what's needed).

Reply:
done